packages/components/src/Datalist/Datalist.component.js

```diff
@@ -78,6 +78,10 @@ function findEntry(titleMap, attributeName, attributeValue = '') {
 }
 
 function getEntryFromName(titleMap, name, restricted) {
+	if (name === '') {
+		return { name: '', value: '' };
```

Comment:
you could return name directly

Reply:
The format of the titlemap entries is `{name, value}`

Reply:
sorry, I meant `{name,value:''}`

Reply:
```suggestion
		return { name, value: '' };
```